pandas/tests/io/test_sql.py

```diff
@@ -875,7 +874,7 @@ def test_timedelta(self):
 
         # see #6921
         df = to_timedelta(Series(["00:00:01", "00:00:03"], name="foo")).to_frame()
-        with tm.assert_produces_warning(UserWarning):
+        with pytest.warns(UserWarning):
```

Comment:
i would leave this as we have a pre-commit hool to avoid using `pytest.warns` and instead use `tm.assert_produces_warning`